smithy-swift-codegen/src/test/kotlin/mocks/MockHttpResponseBindingErrorGenerator.kt

```diff
@@ -34,4 +34,8 @@ class MockHttpResponseBindingErrorGenerator : HttpResponseBindingErrorGeneratabl
             }
         }
     }
+
+    override fun renderServiceError(ctx: ProtocolGenerator.GenerationContext) {
+        // Intentionally empty
```

Comment:
Can you say more why this should be empty?

Reply:
Discussed offline.